importer/KeEn.py

```diff
@@ -0,0 +1,69 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class KeEn(Monument):
+
+    def update_descriptions(self):
+        descs = {"en": "National Monument of Kenya"}
+        for lang in descs:
+            self.add_description(lang, descs[lang])
+
+    def update_labels(self):
+        english = utils.remove_markup(self.name)
+        self.add_label("en", english)
+
+    def set_heritage_id(self):
+        self.add_statement("wlm_id", str(self.id))
+
+    def set_monuments_all_id(self):
+        self.monuments_all_id = str(self.id)
+
+    def exists_with_monument_article(self, language):
+        return super().exists_with_monument_article("en", "monument_article")
+
+    def set_heritage_start(self):
+        if self.has_non_empty_attribute("gazette"):
+            heritage = self.mapping["heritage"]["item"]
+            date_dict = utils.date_to_dict(self.gazette, "%Y")
```

Comment:
Are all of these simple years? if not what happens? Can we add a try/if and add to report on failiure?

Reply:
Yes, they're always simple years. Since the whole list comes from one page, it can be verified visually: https://en.wikipedia.org/wiki/List_of_sites_and_monuments_in_Kenya